src/main/scala/io/moia/protos/teleproto/Writer.scala

```diff
@@ -173,11 +172,6 @@ object Writer extends LowPriorityWrites {
                                              valueWriter: Writer[MV, PV],
                                              ordering: Ordering[PK]): Writer[TreeMap[MK, MV], Map[PK, PV]] =
     (model: TreeMap[MK, MV]) => for ((key, value) <- model) yield (keyWriter.write(key), valueWriter.write(value))
-
-  private class Mapped[M, P, Q](wrapped: Writer[M, P], f: P => Q) extends Writer[M, Q] {
-
-    def write(model: M): Q = f(wrapped.write(model))
-  }
```

Comment:
This `private class` isn't used anywhere in the project, so the newly enforced linting wants it removed.
That breaks the binary compatibility.
Let's discuss if removing this is worth releasing a `v2` or if I shall revert and add a `@nowarn` annotation.

Reply:
I private class couldn't have been used by Scala code but it could have by Java (because it's probably not private in the Bytecode). I'd sat add the MiMa exclusion with a comment that it was private.

Reply:
Added the exclusion in https://github.com/moia-dev/teleproto/pull/97/commits/58b5a2fa87d6b41ab909f44175f73282cbd25a8b.